Tasks/Task2/query04.sql

```diff
@@ -0,0 +1,16 @@
+--Query 4
+SELECT 
+    P.BusinessEntityID,
+    P.LastName,
+    P.FirstName
+FROM AdventureWorks2019.Person.Person P
```

Comment:
Nit: no need to specify db here (eg it used to be in code of conduct; also could cause issues when there are couple dbs sharing same code eg test/uat/prod)